Makefile

```diff
@@ -44,22 +44,26 @@ CLI_VERSION = $(VERSION)
 endif
 CLI_VERSION_PACKAGE = github.com/NVIDIA/k8s-device-plugin/internal/info
 
-GOOS ?= linux
-
 binaries: cmds
 ifneq ($(PREFIX),)
 cmd-%: COMMAND_BUILD_OPTIONS = -o $(PREFIX)/$(*)
 endif
+ifneq ($(shell uname),Darwin)
```

Comment:
question: Is this conditional only applicable to Darwin and Linux or does it include other OSes like Windows, FreeBSD etc?